CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Fixed
+Documentation for supported CycloneDX file formats
```

Comment:
No need for a changelog entry here, but if there was this would be missing the `-`.